org.free_astro.ser_player.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "org.free_astro.ser_player",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "6.8",
+	"sdk": "org.kde.Sdk",
+	"command": "ser-player",
+	"cleanup": [
+		"/include",
+		"*.a",
+		"*.la",
+		"*.cmake"
+	],
+	"finish-args":
+	[
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
```

Comment:
probably need `--device=dri`

Reply:
Application doesn't use OpenGL and current test build is working fine.

Reply:
if it is Qt it does.